modules/persistent-storage-csi-cloning-provisioning.adoc

```diff
@@ -58,5 +56,31 @@ A new PVC `pvc-1-clone` is created.
 ----
 $ oc get pvc pvc-1-clone
 ----
+
+To use the newly cloned PVC to configure a Pod:
```

Comment:
This seems odd to have a second procedure in here like this. It should either be a step in the above procedure (if it's truly necessary to accomplish that task), or it should be a separate procedure.

Reply:
Agree. Originally, it was a separate procedure but the feedback was that it made more sense as part of this one. I've changed this so that the intro sentence is now a validation statement for the previous step, and I've made "Create and save..." the next step.